Astro Error with Docker WSL on Windows

Hi! I'm running into an issue with starting up the frontend container. I've attached a log snippet.

```
2025-01-03 11:50:48 Error: Cannot find module '/app/node_modules/astro/astro.js'
2025-01-03 11:50:48     at Module._resolveFilename (node:internal/modules/cjs/loader:1140:15)
2025-01-03 11:50:48     at Module._load (node:internal/modules/cjs/loader:981:27)
2025-01-03 11:50:48     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:128:12)
2025-01-03 11:50:48     at node:internal/main/run_main_module:28:49 {
2025-01-03 11:50:48   code: 'MODULE_NOT_FOUND',
2025-01-03 11:50:48   requireStack: []
2025-01-03 11:50:48 }
```
![image](https://github.com/user-attachments/assets/f49e1b51-81bd-4c4b-95f1-ac7b94f0dc91)

I'm running the typical setup for Docker on Windows with WSL. I'm following the instructions under Linux using Docker Compose.

--- a/docker-compose.yaml
+++ b/docker-compose.yaml
@@ -9,9 +9,9 @@ services:
     image: dweam_frontend:local
     ports:
       - "127.0.0.1:4321:4321"
-    volumes:
-      - ./dweam_web:/app
-      # - /app/node_modules
+    # volumes:
+    #   - ./dweam_web:/app
+    #   - /app/node_modules
     environment:
       - INTERNAL_BACKEND_URL=http://backend:8080
     # command: npm run dev --host 0.0.0.0

